### PR TITLE
Update guzzle dependency to guzzle 5.

### PR DESCRIPTION
**Pull request by chadicus**
This is simply a rebased copy of #25  It appeared that @pocallaghan just needed to resolve conflicts in order to get this merged.

The guzzle 3 project is deprecated, upgraded to depend on guzzle 5
instead.

### CODE REVIEW

**Comment by m1guelpf** [comment]
Updates?

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ php:
   - 5.6
   - hhvm
 
+matrix:
```
Comment (fezfez) [inline .travis.yml]: If it come with major version, 5.3 can also be removed

**Comment by GrahamCampbell** [comment]
Guzzle 6 would be the way to go. Guzzle 5 is basically dead at this point too.

**Comment by chadicus** [comment]
@GrahamCampbell agreed. I'm working on a branch for that, but I don't fully understand phpspec. When I get the tests running, i'll submit a PR

**Comment by chadicus** [comment]
@GrahamCampbell  I decided to submit the guzzle 6 PR as a work in progress.

**Pull request closed without merging**